client/deis.py

```diff
@@ -1043,18 +1043,35 @@ def certs_add(self, args):
         """
         Binds a certificate/key pair to an application.
 
-        Usage: deis certs:add <cert> <key>
+        Usage: deis certs:add <cert> <key> [--subject-alt-name=<san>...] [options]
```

Comment:
:+1: If that's an easy change, I'd recommend that. I have a feeling the multiple flags will get unruly very quickly.